sources/packages/web/src/types/contracts/StudentContract.ts

```diff
@@ -27,13 +28,13 @@ export interface StudentApplication {
 }
 
 /**
- * DTO object which is returned by student restriction API.
+ * Student restriction DTO.
+ * This object is returned by controller.
+ * Here the key is the restrictionCode and value
+ * is the respective notificationType.
  */
-export interface StudentRestrictionStatus {
-  hasRestriction: boolean;
-  hasFederalRestriction: boolean;
-  hasProvincialRestriction: boolean;
-  restrictionMessage: string;
+export class StudentRestrictionAPIOutDTO {
```

Comment:
This DTO can be interface. (I remember doing the same thing 😊)

Reply:
ya...I noticed that.. after sending the PR request. updated 😄